specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/scenarios/bastionHost.yaml

```diff
@@ -0,0 +1,49 @@
+scope: ResourceGroup
+
+prepareSteps:
+  - step: VirtualNetworks_CreateWithSubnet
+    exampleFile: ../examples/VirtualNetworkCreateSubnet.json
+    resourceUpdate:
```

Comment:
resourceUpdate has been deleted from v1.2 schema, please use requestUpdate instead.

Reply:
If you use schema in the file, so you can have intellisense in VS Code or VS, so you can find the available keywords.
```
# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/documentation/api-scenario/references/v1.2/schema.json
```